src/main/kotlin/model/HMPPSDomainEvents.kt

```diff
@@ -0,0 +1,57 @@
+package uk.gov.justice.hmpps.architecture
+
+import com.structurizr.model.*
+import com.structurizr.view.AutomaticLayout
+import com.structurizr.view.ViewSet
+import uk.gov.justice.hmpps.architecture.annotations.Tags
+
+class DomainEvents private constructor() {
+  companion object : HMPPSSoftwareSystem {
+    lateinit var system : SoftwareSystem
+
+    override fun defineModelEntities(model: Model) {
+
+      system = model.addSoftwareSystem(
+        "hmpps-domain-events",
+        "Domain-wide Event Topic implemented with AWS Simple Notification Service"
+      )
+        .apply {
+          Tags.SOFTWARE_AS_A_SERVICE.addTo(this)
+          Tags.TOPIC.addTo(this)
+        }
+    }
+
+    override fun defineRelationships() {
+    }
+
+    override fun defineViews(views: ViewSet) {
+
+    views.createSystemContextView(
+    system,
+    "hmpps-domain-events-context",
+    "Context view of hmpps-domain-events topic"
+    ).apply {
+      addDefaultElements()
+      enableAutomaticLayout()
+    }
+
+    views.createDeploymentView(
```

Comment:
There are no deployment or container elements for this model, do we need these views?